tests/acceptance/features/environment.py

```diff
@@ -44,9 +46,17 @@ def before_scenario(context, scenario):
     __logger__.info("##############################")
 
     context.expiredusers = ExpiredUsers(TENANT_NAME, USERNAME, PASSWORD)
+    context.createusermanagement = CreateUser()
```

Comment:
Why not context.usermanagement = CreateUser() ? It is something confuse after it the call:

context.createusermanagement.delete_community_users()